### DISCUSSION TITLE
FPGA有中午休息時間嗎? | 統計活著的online board

### DISCUSSION THREAD

**Discussion by henrytsui000** [bol-edu/HLS-SOC-Discussions#17] [General]
如下圖所示
![](https://i.imgur.com/Kil2ntk.jpg)

---

經過多次嘗試，我發現有不少FPGA board不能使用
不如我們大家一起統計可以用的板子來減少學生們快樂踩地雷時光?
**lab2** 
- 或許可以: pynq_10
- 快樂踩地雷: pynq_11, pynq_12

**Comment by tofumanjeff** [comment]
抱歉造成同學困擾，經查最近網路有被攻擊的情況，造成網路壅塞，我們會再進行處理。
另外PYNQ-Z2板子的連線狀況本身就不太穩定，這部分我們之前就有跟生產的廠商聯繫過，他們也在研究造成此問題的原因。
我們這邊已經努力去盡量把影響縮小，例如定時檢查連線狀況反應在租借系統(無法連線會顯示unknown)，以及每日會自動重新開機重置板子，但難免還是會有無法連線的情況，只能請同學多嘗試看看，還請同學見諒。

**Comment by deyulian** [comment]
LAb2-1
pynq_01 也應該有問題
剛剛跑不會過一直報錯 Connection timed out
結果換到pynq_03上就成功了
